db/schema.rb

```diff
@@ -51,6 +51,7 @@
     t.bigint "provider_ids", default: [], array: true
     t.integer "current_recruitment_cycle_year"
     t.boolean "withdrawn_or_declined_for_candidate_by_provider"
+    t.string "rejection_reasons_type"
```

Comment:
could we add this to analytics.yml please to send to BQ, we're putting in a process for this soon 🤞